sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -260,10 +260,13 @@ export class CASService {
       const casKnownErrors = error.response.data[
         CAS_RETURNED_MESSAGES
       ] as string;
+      const casKnownErrorArray = casKnownErrors.split("|").map((error) => {
+        return error.trim();
+      });
```

Comment:
Not a blocker, it can be simplified as below.
```ts
const casKnownErrorArray = casKnownErrors
        .split("|")
        .map((error) => error.trim());`
```